cyclonedx/output/xml.py

```diff
@@ -140,6 +140,73 @@ def _get_component_as_xml_element(self, component: Component) -> ElementTree.Ele
+            if component.get_release_notes().get_description():
+                ElementTree.SubElement(release_notes_e,
+                                       'description').text = component.get_release_notes().get_description()
+            if component.get_release_notes().get_timestamp():
+                ElementTree.SubElement(release_notes_e,
+                                       'timestamp').text = component.get_release_notes().get_timestamp().isoformat()
+            if component.get_release_notes().get_aliases():
+                release_notes_aliases_e = ElementTree.SubElement(release_notes_e, 'aliases')
+                for alias in component.get_release_notes().get_aliases():
+                    ElementTree.SubElement(release_notes_aliases_e, 'alias').text = alias
+            if component.get_release_notes().get_tags():
+                release_notes_tags_e = ElementTree.SubElement(release_notes_e, 'tags')
+                for tag in component.get_release_notes().get_tags():
+                    ElementTree.SubElement(release_notes_tags_e, 'tag').text = tag
+            if component.get_release_notes().get_resolves():
+                release_notes_resolves_e = ElementTree.SubElement(release_notes_e, 'resolves')
+                for issue in component.get_release_notes().get_resolves():
+                    issue_e = ElementTree.SubElement(
+                        release_notes_resolves_e, 'issue', {'type': issue.get_classification().value}
+                    )
+                    if issue.get_id():
+                        ElementTree.SubElement(issue_e, 'id').text = issue.get_id()
+                    if issue.get_name():
+                        ElementTree.SubElement(issue_e, 'name').text = issue.get_name()
+                    if issue.get_description():
+                        ElementTree.SubElement(issue_e, 'description').text = issue.get_description()
+                    if issue.get_source_name() or issue.get_source_url():
+                        issue_source_e = ElementTree.SubElement(issue_e, 'source')
+                        if issue.get_source_name():
+                            ElementTree.SubElement(issue_source_e, 'name').text = issue.get_source_name()
+                        if issue.get_source_url():
+                            ElementTree.SubElement(issue_source_e, 'url').text = str(issue.get_source_url())
+                    if issue.get_references():
+                        issue_references_e = ElementTree.SubElement(issue_e, 'references')
+                        for reference in issue.get_references():
+                            ElementTree.SubElement(issue_references_e, 'url').text = str(reference)
+            if component.get_release_notes().get_notes():
+                release_notes_notes_e = ElementTree.SubElement(release_notes_e, 'notes')
+                for note in component.get_release_notes().get_notes():
```

Comment:
method is getting to complex. put the body of the for/loop in an extra (private) method?